src/aioquic/asyncio/client.py

```diff
@@ -71,14 +75,15 @@ async def connect(
         configuration=configuration, session_ticket_handler=session_ticket_handler
     )
 
-    # connect
+    # explicitly enable IPv4/IPv6 dual stack
     sock = socket.socket(socket.AF_INET6, socket.SOCK_DGRAM)
     try:
         sock.setsockopt(socket.IPPROTO_IPV6, socket.IPV6_V6ONLY, 0)
         sock.bind((local_host, local_port))
-    except:
+    except BaseException:
```

Comment:
`Exception` not `BaseException`

Reply:
Socket should be closed when any exceptions occur, so `Exception`, in my view, is not an ideal workaround.